src/main/java/com/study/realworld/user/presentation/UserController.java

```diff
@@ -39,4 +42,10 @@
         User user = userService.login(request.toModel());
         return ResponseEntity.ok().body(UserResponse.createResponse(user, tokenProvider.createToken(user)));
     }
+
+    @GetMapping(value = "/user")
+    public ResponseEntity<UserResponse> getLoginUser(@AuthenticationPrincipal Long userId) {
+        User user = userService.getLoginUser(userId);
+        return ResponseEntity.ok().body(UserResponse.createResponse(user, SecurityUtil.getAccessToken()));
```

Comment:
메서드안에서 `메서드 호출` -> `메서드 호출`이여서   
차라리 변수로 따로 빼서 넣으면 좋을 것 같아요  
```suggestion
        var token = SecurityUtil.getAccessToken(); // 타입을 몰라서 var로 넣었습니다. 
        return ResponseEntity.ok().body(UserResponse.createResponse(user, token));
```

Reply:
파라미터에 분기문만 안 쓴다면 괜찮지 않을까요?
메서드 -> 메서드호출 -> 메서드호출이라 depth가 깊어서 그러신걸까용??

Reply:
네네 depth 가 깊어져서 3자 입장에서 가독성을 조금 해치는 것 같아서요!

Reply:
자동완성 사용하면 타입 저절로 나올 것 같아!(컨트롤 알트 브이였나 컨트롤 옵션 브이였나..)